tools/java/src/main/java/org/unicode/cldr/json/Ldml2JsonConverter.java

```diff
@@ -973,7 +967,7 @@ private void resolveArrayItems(JsonWriter out,
         ArrayList<CldrNode> nodesForLastItem,
         ArrayList<CldrItem> arrayItems)
         throws IOException, ParseException {
-        boolean rbnfFlag = false;
+//        boolean rbnfFlag = false;
```

Comment:
Delete rather than comment out? Recent best practice for "clean code" is "no commented-out code"

Reply:
Same below